pegasus/sites.v3/code.org/public/student/middle-high.haml

```diff
@@ -2,71 +2,191 @@
 title: Computer Science Curriculum for Grades 6-12
 theme: responsive
 ---
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/student/school-styles.css", rel: "stylesheet"}
 
-:css
-  h2 {
-    margin: 0;
-  }
 
 - js_locale = request.locale.to_s.downcase.tr('-', '_')
 %script{src: webpack_asset_path("js/#{js_locale}/common_locale.js")}
 %script{src: webpack_asset_path('js/code.org/public/student/middle-high.js')}
 %link{href: "/shared/css/course-blocks.css", rel: "stylesheet", type: "text/css"}
 = view 'mobilecheck.js'
```

Comment:
Delete lines `9-13` (re: [this comment](https://github.com/code-dot-org/code-dot-org/pull/54325/files#r1369265291))